src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -127,6 +147,14 @@ IFACEMETHODIMP_(void) FancyZones::Run() noexcept
         SetThreadDpiAwarenessContext(DPI_AWARENESS_CONTEXT_UNAWARE);
         SetThreadDpiHostingBehavior(DPI_HOSTING_BEHAVIOR_MIXED);
     }}).wait();
+
+    if (RegOpenKeyEx(HKEY_CURRENT_USER, L"Software\\Microsoft\\Windows\\CurrentVersion\\Explorer\\VirtualDesktops", 0, KEY_ALL_ACCESS, &m_virtualDesktopsRegKey) != ERROR_SUCCESS) {
+        return;

```

Comment:
Instead of returning if `RegOpenKeyEx` failed, do the opposite, and call the code at lines 155-157 if `RegOpenKeyEx` succeeds.